src/animate.jl

```diff
@@ -24,19 +24,19 @@ function animate(vis::MechanismVisualizer,
         q = interpolated_configurations(t)
         set_configuration!(state, q)
         rbd.normalize_configuration!(state)
-        _render_state!(vis)
+        _render_state!(mvis)
         framenum += 1
         t == tf && break
     end max_rate = fps
 end
 
 function MeshCat.Animation(mvis::MechanismVisualizer,
-        times::AbstractVector{<:Real},
-        configurations::AbstractVector{<:AbstractVector{<:Real}};
-        fps::Integer=30)
+                           times::AbstractVector{<:Real},
+                           configurations::AbstractVector{<:AbstractVector{<:Real}};
+                           fps::Integer=30)
     @assert axes(times) == axes(configurations)
     interpolated_configurations = interpolate((times,), configurations, Gridded(Linear()))
-    animation = Animation()
+    animation = Animation(mvis.visualizer["/meshcat"])
```

Comment:
I found this is one way of getting `vis` here, but I think I prefer explicitly adding a new parameter to the signature for passing `vis` directly (no hidden "tricks").